<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="15" hostname="DESKTOP-DJ4FDME" failures="3" tests="3" name="Test(failed)" time="0.135" errors="0" timestamp="2024-08-20T06:20:04 IST">
  <testcase classname="demoTestNG.TestScenario9" name="sixth_testcase" time="0.033">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario9.sixth_testcase(TestScenario9.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="first_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.copy.TestScenario.first_testcase(TestScenario.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="fifth_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario4.fifth_testcase(TestScenario4.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.TestScenario9" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario9" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario9" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario9" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario9" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.copy.TestScenario" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
</testsuite> <!-- Test(failed) -->
